tools/ncpms-api-crawler/ncpms-crawler.py

```
@@ -0,0 +1,162 @@
+        try:
+            data = json.loads(text)
+            svc = data.get("service", {})
+            info["ecologyInfo"] = remove_html_tags_from_api_text(svc.get("ecologyInfo", ""))
+            info["biologyPrvnbeMth"] = remove_html_tags_from_api_text(svc.get("biologyPrvnbeMth", ""))
+            info["chemicalPrvnbeMth"] = remove_html_tags_from_api_text(svc.get("chemicalPrvnbeMth", ""))
+            info["preventMethod"] = remove_html_tags_from_api_text(svc.get("preventMethod", ""))
+        except:
+            pass
+    return info
+
+def main():
```

Ruff (0.15.10)

[warning] Too many branches (16 > 12)

(PLR0912)